layouts/partials/hooks/body-end.html

```diff
@@ -26,3 +26,24 @@
 }
 </script>
 {{- end -}}
+{{- if .HasShortcode "code" -}}
```

Comment:
This would work better as a logical OR condition: add the script just once, if _either_ of the shortcodes is present.

Reply:
Yes. I missed that. Updated.